### PR TITLE
verible: 0.0.3253 -> 0.0.3410

### PR DESCRIPTION
**Pull request by hzeller**
## Description of changes

Update to latest; a few improvements in the language server and some bugs fixed.

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by hzeller** [comment]
(waiting for the aarch64 build to run to see the expected fetchAttrs sha256)

File: pkgs/development/tools/language-servers/verible/default.nix [hunk 1]
```diff
@@ -1,11 +1,11 @@
-{ lib
```
Comment (newAM) [inline pkgs/development/tools/language-servers/verible/default.nix:1]: The nix convention for these is to go in the order they appear in the file, with exception of `lib` that goes first: https://ryantm.github.io/nixpkgs/contributing/coding-conventions/

**Review by newAM — APPROVED** [review]
Builds and runs on x86_64-linux

File: pkgs/development/tools/language-servers/verible/default.nix (hunk 1 above)
Comment (hzeller) [reply pkgs/development/tools/language-servers/verible/default.nix:1]: Ah, interesting, good to know. In another review, the reviewer asked me to sort these. Will have to read the documentation more throuroughly.

Anyway, fixed now.

**Review by newAM — APPROVED** [review]

**Comment by hzeller** [comment]
Adding @NickCao who merged [the last update]( https://github.com/NixOS/nixpkgs/pull/232310)

**Review by NickCao — APPROVED** [review]

**Pull request merged**